doc/examples/segment_extending_clustering_framework.py

```diff
@@ -4,7 +4,7 @@
 ==========================================================
 
 QuickBundles [Garyfallidis12]_ is a flexible algorithm that requires only a
-distance metric and an adjacency threshold to perform clustering. There is a
+distance metric and an adjacenct threshold to perform clustering. There is a
```

Comment:
The term is accurate and matches the term used in the original paper and cross-referenced above:
https://www.frontiersin.org/articles/10.3389/fnins.2012.00175/full

Reply:
Firstly, Thank you @jhlegarreta for reviewing the PR and correcting me about "adjacency" I didn't really know the term but now I have changed it.

Secondly, I had made a PR https://github.com/dipy/dipy/pull/2094 regarding making the examples more simple and beginner friendly in a notebook manner but then @skoudoro is working on a sphinx format for the tutorials. Since these are simple python scripts and people run them directly it is good to place all imports in the beginning, plus it is just more systematic. The use of these modules is explained in the comments right where the classes are so it should not be an issue.

And lastly please stop using "I'd dare to" in every review you make cause we are all here as a community to contribute and not to challenge each other.

Reply:
Hi @ShrishtiHore ! Thank you for this PR 👍 I don't think @jhlegarreta is trying to be aggressive whatsoever. In fact, the phrase `I'd dare to say` is a rather polite way of giving constructive criticism on your PR, given that its is not his research or contribution to the code-base. 

However, I do see how this could be misconstrued as impolite and appreciate you sharing the way it made you feel. We are more than happy to have you on board and I believe we are looking at your PR's very actively! Looking forward to more contributions and engagement with the community!

Reply:
Hi@ShreyasFadnavis Thank you I totally understand. And I don't mind about it now. I would surely love to engage more with the community.

Reply:
Just a side note, because just from reading the emails threads of the notifications for some of those PRs, I have felt some friction about language and the way to address each other.

I just want everyone to remember that we all have our native language, and more often than not, it is not english. If someone sometimes feel that another member is being rude, most of the time it simply is a misunderstanding due to varying language patterns.

We're all in this together, and especially in those weird times where we are now, we should strive to be caring for each other.

Reply:
Hi @jchoude Thank you and I really appreciate that you took time to reflect.